examples/scope.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) NVIDIA
```

Comment:
I'm pretty sure you are free to use your own copyright for code that you wrote, as long as it's all under the same license.